lib/Dialect/SSP/SSPOps.cpp

```diff
@@ -19,6 +19,47 @@
 using namespace circt;
 using namespace circt::ssp;
 
+//===----------------------------------------------------------------------===//
+// InstanceOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult InstanceOp::verify() {
+  auto &ops = getBodyBlock()->getOperations();
+  auto it = ops.begin();
+  if (ops.size() != 2 || !isa<OperatorLibraryOp>(*it) ||
+      !isa<DependenceGraphOp>(*(++it)))
+    return emitOpError()
+           << "must contain exactly one 'library' op and one 'graph' op";
```

Comment:
nit: misleading error message in case `DependenceGraphOp` comes before `OperatorLibraryOp`. Additionally, as I understand, SSP passes won't need a strict ordering between these two ops, so enforcing it might be overly restrictive.

Reply:
> nit: misleading error message in case `DependenceGraphOp` comes before `OperatorLibraryOp`.

Fixed.

> Additionally, as I understand, SSP passes won't need a strict ordering between these two ops, so enforcing it might be overly restrictive.

Yes, semantically it doesn't make a difference, but this ordering constraint ensures that I can roundtrip an instance via the scheduling infra in about 2-3 PRs -- with only the `scheduling::Problem`, I wouldn't know if the `library` or `graph` op came first.